client/my-sites/exporter/guided-transfer-card/index.jsx

```diff
@@ -62,7 +64,9 @@ class GuidedTransferCard extends Component {
 							{ translate( 'Guided Transfer' ) }
 						</h1>
 						<h2 className="guided-transfer-card__subtitle">
-							<span className="guided-transfer-card__price">$129</span>
+							<span className="guided-transfer-card__price">
+								{ cost }
+							</span>
 							&nbsp;
 							{ translate( 'One-time expense' ) }
```

Comment:
This is not a huge deal and probably doesn't really matter but I thought about it nonetheless.

I know these were separate before but I'm wondering if having the translation include the cost will give more context to the person doing translations.

I feel like translating `%(cost)s{{nbsp/}}One-time expense` might have a different translation than `One-time expense` in some other language.